external/llvm-project/llvm/CMakeLists.txt

```diff
@@ -2,6 +2,12 @@
 
 cmake_minimum_required(VERSION 3.13.4)
 
+# CMP0116: Ninja generators transform `DEPFILE`s from `add_custom_command()`
+# New in CMake 3.20. https://cmake.org/cmake/help/latest/policy/CMP0116.html
```

Comment:
Will find time for looking at the rest of the changes later, but a quick comment: I think a better idea is to now install cmake 3.15.1 (and upgrade when upstream upgrade) instead of using a newer cmake than upstream and adopt hack like this.

Reply:
This is literally copied from llvm itself

Reply:
Ahh gotcha, NVM then!